ush/ufsda/stage.py

```diff
@@ -1,19 +1,94 @@
 from r2d2 import fetch
 from solo.basic_files import mkdir
-from solo.date import Hour, DateIncrement
+from solo.date import Hour, DateIncrement, date_sequence
 from solo.logger import Logger
 from solo.stage import Stage
 from solo.configuration import Configuration
 from solo.nice_dict import NiceDict
+from datetime import datetime, timedelta
 import os
 import shutil
 from dateutil import parser
 import ufsda
 import logging
 import glob
+import xarray
+import sys
+import numpy as np
+
+# TODO: There's probably a better way to do this
+jedi_lib = os.getenv('JEDI_LIB')
+sys.path.append(os.path.join(jedi_lib, 'pyiodaconv'))
+sys.path.append(glob.glob(os.path.join(jedi_lib, 'python3.*/pyioda'))[0])
```

Comment:
All of this is fine for temporary, but you should be aware, that `stage` is likely going to become generic and not R2D2y soon... I think we are probably going to have to talk about R2D2's role here going forward

Reply:
@CoryMartin-NOAA , I know we said I should just add the paths of `pyioda` and `pyiodaconv` to `PYTHONPATH`, but I can't figure out how to do this for the ctests using the `cmake_generator_expression` `TARGET_FILE_DIR`, which is what's used in `iodaconv` ... The issue is that the path is python version dependent, so the path to pyioda isn't resolved at config time ... Asking for a pass on that issue ... Pretty please?

Reply:
I see... well that's a problem. Can you use the cmake variable `Python_VERSION` combined with what is here:
https://github.com/NOAA-EMC/GDASApp/blob/38ca1e9267507cea9ec42f2b50045a2706a0876d/test/atm/CMakeLists.txt#L25

If not, then I will give you a pass.

Reply:
maybe you will want `Python_VERSION_MAJOR`

Reply:
Dam ... yes, that should work. ```Python_VERSION_MINOR``` returns the `9` ...